storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/TransferManager.java

```diff
@@ -117,7 +121,7 @@
                                 progressLock, totalProgress);
 
                     final String blockId = Base64.getEncoder().encodeToString(
-                            UUID.randomUUID().toString().getBytes());
+                            UUID.randomUUID().toString().getBytes(StandardCharsets.UTF_8));
```

Comment:
Could use UTF_8 instead of StandardCharsets.UTF_8
"import static java.nio.charset.StandardCharsets.UTF_8;"

Reply:
Ok with either approach. But to standardize the way of static imports better to put it into checkstyle. I will change as you suggested.